Gaijinizer patch/Map059.txt

```diff
@@ -2,62 +2,65 @@
 > BEGIN STRING
 目ぇ覚めたか？\\N[1]君。
 > CONTEXT: Map059/events/1/pages/0/14/Dialogue/other3
-Did you wake up? \\N[1] you.#MTLed
+Are you awake, \\N[1]-kun?#Ron
 > END STRING
 
 > BEGIN STRING
 ほれ、外の自販機で買ってきた缶コーヒー。
 
 これなら、安心して飲めるじゃろ。
 > CONTEXT: Map059/events/1/pages/0/24/Dialogue/other3
-Well, I bought a canned coffee at an outside
-vending machine. With this, you can drink with
-confidence.#MTLed
+Here, I bought this canned coffee from a vending
+machine outside. If it's from a vending machine
+you won't have to worry about what's in it so
+drink up.#Ron
 > END STRING
 
 > BEGIN STRING
 あ、ありがとう……
 > CONTEXT: Map059/events/1/pages/0/28/Dialogue/kazuya
-Oh, thank you ......#MTLed
+T- thanks...#Ron
 > END STRING
 
 > BEGIN STRING
 ………あ、あの転送装置な、
 
  転送されるとひどく酔うっちうデメリットがあっての。
 > CONTEXT: Map059/events/1/pages/0/36/Dialogue/other3
-......Oh, that transfer device,
-I have a limitation that makes me drunk heavily
-when transferred.#MTLed
+... T- that transfer device...
+
+It can make you pretty nauseous when you use it.#Ron
 > END STRING
 
 > BEGIN STRING
 ほんの短距離でも、ご覧のとおり気絶するレベルやから
 
 実用化に調整を進めてた試作品だったんよ……
 > CONTEXT: Map059/events/1/pages/0/40/Dialogue/other3
-Even a short distance, as you see from the
-stunning levels and. It was a prototype that had
-been adjusted for commercialization ...#MTLed
+As you now know, even very short distance travel's
+enough to knock you off your feet. It's a
+prototype I've been working on with an eye to
+commercialize...#Ron
 > END STRING
 
 > BEGIN STRING
 でも今回は、緊急事態やったしな……
 
 あれ以上待ったら、きっとお前さん………
 > CONTEXT: Map059/events/1/pages/0/44/Dialogue/other3
-But this time, I did an emergency situation ...
-If you wait more than that, surely you .........#MTLed
+But since you were really in a pinch I had to use
+it. If I'd waited any longer, you would've been
+done for...#Ron
 > END STRING
 
 > BEGIN STRING
 言い逃れできる状況でも無かし、正直に話すわ。
 
 …シェルターで目覚めた時から、お前さんを見てた。
 > CONTEXT: Map059/events/1/pages/0/52/Dialogue/other3
-I will not speak even if I can escape it and
-speak honestly....I watched you since I woke up
-at the shelter.#MTLed
+I'm not gonna give you the runaround, I'll be
+honest with you. I've been watching you since you
+woke up in the shelter.#Ron
```

Comment:
Flow:
>_**I'm not gonna give you the runaround.
 I'll be honest with you: I've been watching you
since you woke up in the shelter.**_